Description: Replace "#!/usr/bin/env perl" by "#!/usr/bin/perl -w"
Origin: Norbert Preining <preining@debian.org>
Forwarded: no
Author: Norbert Preining <preining@debian.org>
Last-Update: Mon, 13 Aug 2018 09:29:28 +0900

---
 bin/latexml     |    2 +-
 bin/latexmlc    |    2 +-
 bin/latexmlfind |    2 +-
 bin/latexmlmath |    2 +-
 bin/latexmlpost |    2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

--- latexml-debian.orig/bin/latexml
+++ latexml-debian/bin/latexml
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl -w
 # /=====================================================================\ #
 # |  latexml                                                            | #
 # | main conversion program                                             | #
--- latexml-debian.orig/bin/latexmlc
+++ latexml-debian/bin/latexmlc
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl -w
 # /=====================================================================\ #
 # |  latexmlc                                                           | #
 # | client/server conversion program                                    | #
--- latexml-debian.orig/bin/latexmlfind
+++ latexml-debian/bin/latexmlfind
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl -w
 # /=====================================================================\ #
 # |  latexmlfind                                                        | #
 # | xml search utility program                                          | #
--- latexml-debian.orig/bin/latexmlmath
+++ latexml-debian/bin/latexmlmath
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl -w
 # /=====================================================================\ #
 # |  latexmlmath                                                        | #
 # | math conversion program                                             | #
--- latexml-debian.orig/bin/latexmlpost
+++ latexml-debian/bin/latexmlpost
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl -w
 # /=====================================================================\ #
 # |  latexmlpost                                                        | #
 # | post processing conversion program                                  | #
